collection/create_collection.py

```diff
@@ -23,38 +25,83 @@
 
 config = load_config(".config/config.json")
 
+# TODO: add arg parser for populate from snapshot or not
+parser = argparse.ArgumentParser(description="Create a Qdrant collection from BigQuery")
+
+# Add a boolean argument that defaults to True.
+# Use 'store_false' to set the value to False when the flag is present.
+parser.add_argument(
+    "-rs",
+    "--restore-from-snapshot",
+    action="store_true",  # This will set the value to False when the flag is used
+    default=False,  # Default value is False
+    dest="restore_from_snapshot",
+    help="Set to True to disable restoring from a snapshot. Defaults to False.",
+)
+
+args = parser.parse_args()
+
+client = load_qdrant_client(QDRANT_HOST, port=QDRANT_PORT)
 
 all_query_read = query_all_feedback.replace("@PUBLISHING_VIEW", str(PUBLISHING_VIEW))
 
 eval_query_read = query_labelled_feedback.replace(
     "@LABELLED_FEEDBACK_TABLE", str(LABELLED_FEEDBACK_TABLE)
 ).replace("@PUBLISHING_VIEW", str(PUBLISHING_VIEW))
 
-print("Reading data from BigQuery...")
+# Create feedback and eval collections
+for name, query in [
+    # (COLLECTION_NAME, all_query_read),
```

Comment:
Is this a scenario where we uncomment line 54 if we want to use the whole collection, or use line 55 for just the evaluation collection?